### ISSUE TITLE
Implement HTTPS connections for view

### ISSUE THREAD

**Issue by anthonyjchriste** [openpowerquality/opq#124]
Currently view on emilia only provides an http end point. That means that logins send the username and password in the clear to our servers presenting possible avenues for attack. 

I believe we should investigate how to setup proper security for OPQView.

Please do work for this task in a branch called issue-124.

**Comment by philipmjohnson** [comment]
Yes, https is offered with a single click on most hosting services (Galaxy, Netlify, GitHub) using Let's Encrypt. We should talk with ITS about whether they can facilitate https.
